features/admin.py

```diff
@@ -77,13 +77,13 @@ class FeatureAdmin(TranslatableAdmin, admin.OSMGeoAdmin):
     list_display = (
         "ahti_id",
         "name",
-        "category",
+        #    "category",
```

Comment:
do not leave commented out code 🙂 if it's not needed anymore, just delete the lines